plugin/utils/tools.py

```diff
@@ -68,6 +69,17 @@ def run_command(command, shell=False, cwd=path.curdir, env=environ,
                 "Executable file not found executing: {}".format(command))
         return output_text
 
+    @staticmethod
+    def run_command_with_logging(command, cwd=path.curdir):
```

Comment:
Why do we need this function?

Reply:
xcpretty parses output from xcodebuild, and to get commands for every file you should rebuild whole project. That's a long process, and to see what's going on and to be sure that all is fine it's better to print logs in real time.

Reply:
I don't think that the console is the place to print things. If this is interfacing the user, we should think about showing a panel.